docs/vectorizer/api-reference.md

```diff
@@ -617,6 +617,15 @@ You use it to recursively split text into chunks using multiple separators.
 
 A JSON configuration object that you can use in [ai.create_vectorizer](#create-vectorizers).
 
+### Understanding Overlap
+The recursive character text splitter creates overlaps only when splitting within words (using the empty string separator "").
```

Comment:
Is this statement correct? In the following example, the "" splitter is not used, but there is still overlapping text:

```python
text_splitter = RecursiveCharacterTextSplitter(separators=["\n\n", "\n", " ", ""],chunk_size=20,chunk_overlap=10)
text_splitter.create_documents(["Lorem Ipsum is simply dummy text of the printing and typesetting industry."])
```
Produces:
```
['Lorem Ipsum is', 'Ipsum is simply', 'is simply dummy', 'dummy text of the', 'of the printing and', 'and typesetting', 'industry.']
```

Reply:
Your code actually contains the "" splitter. But I tried without it and you're correct that it still produces overlap. I can't quite figure out how this works. Some chunks seems to have overlap and some don't. Maybe linking to langchain is enough.